core/src/main/java/jenkins/ContextClasssLoaderFilter.java

```diff
@@ -0,0 +1,46 @@
+package jenkins;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import java.io.IOException;
+
+/**
+ * {@link Filter} that sets up the context classloader.
+ *
+ * @author Kohsuke Kawaguchi
+ */
+public class ContextClasssLoaderFilter implements Filter {
```

Comment:
typo

Reply:
fixed in 929c9b09ee8eefc903efef99f49441d44aec4b74